tests/kernel/common/src/irq_offload.c

```diff
@@ -35,10 +35,93 @@ static void offload_function(const void *param)
  *
  * @ingroup kernel_interrupt_tests
  *
- * @details Check whether offloaded running function is in interrupt
- * context, on the IRQ stack or not.
+ * @details
+ * Test Objective:
+ * - To verify the kernel architecture layer shall provide a means to raise an
+ *   interrupt from thread code that will run a user-provided C function under
+ *   the interrupt context.
+ * - We check whether offloaded running function is in interrupt context, on
+ *   the IRQ stack or not.
+ *
+ * Testing techniques:
+ * - Interface testing, function and block box testing,
+ *   dynamic analysis and testing
+ *
+ * Prerequisite Conditions:
+ * - CONFIG_IRQ_OFFLOAD=y was set.
```

Comment:
Please remove "was set".

Reply:
Got it fix, thanks. I'll apply this to all other PRs which are like this.